bundles/TinymceBundle/src/PimcoreTinymceBundle.php

```diff
@@ -29,6 +29,12 @@ class PimcoreTinymceBundle extends AbstractPimcoreBundle implements PimcoreBundl
 
     public function getCssPaths(): array
     {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '11.5.0',
+            'TinMCE is deprecated, please use quill instead'
```

Comment:
Maybe some more information *how* to use quill instead?

Reply:
https://github.com/pimcore/pimcore/pull/17884/files#diff-0f5fa6563281919aebddf22fd70fdfa7319eefd6610ade9ae34ade6869c78e25R35
I think this is now explained with the bundle. What is your opinion?